http-endpoint-example/README.md

```diff
@@ -0,0 +1,46 @@
+# How to download Met Office atmospheric model data using an HTTP endpoint
+A demonstration code example of downloading objects in response to SNS messages via an HTTP endpoint.
+
+By using the following you will be able to subscribe to an Amazon Web Services (AWS) Simple Notification Service (SNS) topic and start downloading objects.
+The running python script will accept notifications from the SNS topic and download the objects according to the input parameters.
+
+
+## Prerequisites:
+
+  * A service API key
+  * A Python 3 environment containing the "flask" and "requests" libraries
+  * A server with a Public IP or DNS address with an open port. Ports other than standard http/https are not reconmmended as described [here](https://aws.amazon.com/sns/faqs/ "AWS documentation")
+  * If used in production a method of verifying the messages are from AWS, this can be found [here](http://docs.aws.amazon.com/sns/latest/dg/SendMessageToHttp.verify.signature.html "AWS documentation")
+
+# Actions:
+1. ### Clone repo onto the server
+
+2. ### Start listening for notifications
+   Start polling the SNS topic by running host.py with the following command
+````
+  python host.py <port> <api_key> <start_time> <end_time> <comma-separated diagnostic names)
```

Comment:
Have a mismatch between angle and round bracket on the last param '<comma-separated diagnostic names)'. Could maybe consider changing to <comma_separated_diagnostic_names> to keep consistent format?